container/apptabs.go

```diff
@@ -89,6 +88,34 @@ func (t *AppTabs) CurrentTabIndex() int {
 	return t.current
 }
 
+// Enable enables the specified TabItem.
+//
+// Since: 2.2
```

Comment:
2.2 is already released, this would need to be 2.3

Reply:
Also we have a `Disablable` interface with `Enable` which enables the whole widget, so this name overloading may be confusing. I would recommend `EnableItem` to indicate that it's function is different (same with `Disable` etc).

Reply:
Thanks for the review! I believe I've made your suggested updates, and the static analysis is now passing. There is now just the `TestForm_Disabled` test that is sometimes failing, but I believe it's unrelated to this change. The same test fails when I run it prior to my changes e.g. `git checkout 2504bea3c6a5c97a1d5b45d7c3a39128de6fba5a && go test ./...`.

Reply:
There does seem to be an instability in that form test. All others are now fixed but that one remains a mystery

Reply:
I believe I may have solved the issue with the form test: https://github.com/fyne-io/fyne/pull/3175.